frontends/ol-util/assets/card.scss

```diff
@@ -0,0 +1,44 @@
+@use "variables";
+
+%card {
+    width: 100%;
+    background-color: variables.$card-background;
+    border-radius: 5px;
+    margin: 0 0 20px 0;
+  
+    &:hover {
+      box-shadow: 3px 4px 12px 0 rgba(0, 0, 0, 0.36);
+    }
+  
+    .card-contents {
+      margin: 0;
+      padding: 20px;
+    }
+  
+    .card-title {
```

Comment:
This class was originally called `.title`. After taking a pretty thorough look at our CSS rules for `.title`, it's was clear to me that almost all of our `.title` are completely unrelated to card titles. For example, `channels.scss` declares a rule targeting `.title-and-headline .title`, but the only usage of `title-and-headline` is outside of cards, so that rule never affects cards.